word-guess-classes.rb

```diff
@@ -0,0 +1,235 @@
+require 'colorize'
+
+class Word
+  attr_accessor :guessed_letters, :blanks, :wrong_guesses, :letters, :name
+
```

Comment:
I love that the classes are in their own separate file. We hadn't talked about this organization in class yet, but typically you would go even further and have one file per class.